### PR TITLE
Rb v1.1.0

### PR DESCRIPTION
**Pull request by gschmidt-usgs**
Peer review for v1.1.0

### CODE REVIEW

File: espa/metadata/metadata_api.py [hunk 1]
```diff
@@ -22,19 +22,18 @@ class MetadataError(Exception):
...
 class Metadata(XMLInterface):
-    """Enhances the XMLInterface specifically for an ESPA Metadata XML
+    """Enhances the XMLInterface specifically for an XSD Schema defined XMLs
 
     Determines the best source for the schema to use for validation of the
-    specified ESPA Metadata XML.  First the ESPA_SCHEMA environment variable
-    is searched for and used if present.  Second the installation directory
-    where the schema should exist if installed on the local system.  And third
-    the URI where the schema should reside on the internet.
+    specified Metadata XML.  First environment variables are used in this
```
Comment (raydittmeier) [inline espa/metadata/metadata_api.py]: Some rewording is needed.  Remove one of the "this"s in "used in this this order".  Also, the same sentence has "environment variables are used ... and used if present".  One of the uses of "used" probably should be removed or changed to a different word [maybe the first one should be "checked"].  Also, the sentence after that one is incomplete.

File: espa/metadata/metadata_api.py [hunk 2]
```diff
@@ -22,19 +22,18 @@ class MetadataError(Exception):
...
+    this order (XML_SCHEMA, ESPA_SCHEMA, ARD_SCHEMA) and used if present.
+    Second the URI where the schema should reside on the internet.
     """
 
     def __init__(self, xml_filename=None):
         """Object initialization and parsing of the XML document
 
-        Performes determination for the source of the ESPA Metadata XML schema
+        Performes determination for the source of the Metadata XML schema
```
Comment (raydittmeier) [inline espa/metadata/metadata_api.py]: Performes --> Performs

**Review by raydittmeier — APPROVED** [review]
Approved, since the comments were just on comments, but fix them if possible.

**Comment by gschmidt-usgs** [comment]
Changes look good and the ARD metadata support is working well.

File: espa/metadata/metadata_api.py (hunk 2 above)
Comment (ghost) [reply espa/metadata/metadata_api.py]: Fixed spelling

File: espa/metadata/metadata_api.py (hunk 1 above)
Comment (ghost) [reply espa/metadata/metadata_api.py]: Reformatted and reworded that section

**Pull request merged**